### PR TITLE
[azure] add os disk cache check

### PR DESCRIPTION
**Pull request by jinyunma**
Add checkpoint for bug OCPBUGS-33470, also add same in testcase [OCP-21540](https://polarion.engineering.redhat.com/polarion/#/project/OSE/workitem?id=OCP-21540)

### CODE REVIEW

**Comment by jinyunma** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installation-nightly-4.16-azure-ipi-disktype-premium-lrs-disksize-f14

File: ci-operator/step-registry/cucushift/installer/check/azure/cucushift-installer-check-azure-chain.yaml [hunk 1]
```diff
@@ -7,5 +7,6 @@ chain:
   - ref: cucushift-installer-check-azure-cp-lb-probes
   - ref: cucushift-installer-check-azure-cpms
   - ref: cucushift-installer-check-azure-disktype
+  - ref: cucushift-installer-check-azure-diskcache
```
Comment (jianlinliu) [inline ci-operator/step-registry/cucushift/installer/check/azure/cucushift-installer-check-azure-chain.yaml:10]: I am thinking is it good to merge `cucushift-installer-check-azure-disktype` and `cucushift-installer-check-azure-diskcache` as `cucushift-installer-check-azure-disk` for now?

Comment (jinyunma) [reply ci-operator/step-registry/cucushift/installer/check/azure/cucushift-installer-check-azure-chain.yaml]: good suggestion, merged.

**Comment by jinyunma** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installation-nightly-4.16-azure-ipi-disktype-premium-lrs-disksize-f14

File: ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/disktype/disksize/provision/cucushift-installer-rehearse-azure-ipi-disktype-disksize-provision-chain.yaml [hunk 2]
```diff
@@ -8,6 +8,6 @@ chain:
   - ref: enable-qe-catalogsource
   - chain: cucushift-installer-check
   - chain: cucushift-installer-check-azure
-  - ref: cucushift-installer-check-azure-disksize
+  - ref: cucushift-installer-check-azure-disk-size
```
Comment (jianlinliu) [inline ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/disktype/disksize/provision/cucushift-installer-rehearse-azure-ipi-disktype-disksize-provision-chain.yaml:11]: I am thinking if we even can merge this step to `cucushift-installer-check-azure-disk` ?

**Comment by jinyunma** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installation-nightly-4.16-azure-ipi-disktype-premium-lrs-disksize-f14

**Pull request merged**